apps/dokploy/templates/teable/index.ts

```diff
@@ -0,0 +1,42 @@
+import {
+	type Schema,
+	type Template,
+	generateHash,
+	generatePassword,
+	generateRandomDomain,
+} from "../utils";
+
+export function generate(schema: Schema): Template {
+	const mainServiceHash = generateHash(schema.projectName);
+	const password = generatePassword();
+	const randomDomain = generateRandomDomain(schema);
+	const envs = [
+		`TEABLE_HOST=${randomDomain}`,
+		"TEABLE_PORT=3000",
+		`HASH=${mainServiceHash}`,
+		"TIMEZONE=UTC",
+		"# Postgres",
+		"POSTGRES_HOST=teable-db",
+		"POSTGRES_PORT=5432",
+		"POSTGRES_DB=teable",
+		"POSTGRES_USER=teable",
+		`POSTGRES_PASSWORD=${password}`,
+		"# App",
+		"PUBLIC_ORIGIN=https://${TAEBLE_HOST}",
+		"PRISMA_DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}",
+		"PUBLIC_DATABASE_PROXY=127.0.0.1:42345",
```

Comment:
```suggestion
		"PUBLIC_DATABASE_PROXY=${POSTGRES_HOST}:${POSTGRES_PORT}",
```

Reply:
it is the same thing as in previous comment, the URL related to https://help.teable.io/developer/database-connection, it shouldn't be a localhost, but it should have the real domain address